Build/build.ps1

```diff
@@ -166,7 +169,10 @@ function Update-CodeCoveragePercentInTextFile {
         [int] $CodeCoveragePercent = 0,
```

Comment:
Here you can move the parameter to be the last one, as it's not mandatory and the newly added are both mandatory.